test/com/ableton/VirtualEnvTest.groovy

```diff
@@ -125,6 +126,54 @@ class VirtualEnvTest extends BasePipelineTest {
     assertEquals(venv.destDir, createdVenv.destDir)
   }
 
+  @SuppressWarnings('TrailingWhitespace')
+  @Test
+  void createPyenv() throws Exception {
+    String pythonVersion = '1.2.3'
+    String pyenvRoot = '/mock/pyenv/root'
+    helper.registerAllowedMethod('fileExists', [String]) { return true }
+    helper.registerAllowedMethod('isUnix', []) { return true }
+    // Note: The extra empty line is needed here to match the sh mock
+    JenkinsMocks.addShMock("""
+      
```

Comment:
I don't know whether it would work, but something like
```suggestion
      ${String()}
```
would prevent me from having to change the settings of my text editor wrt. trailing spaces when I save this file.
Also it would allow removing the warning suppression.

Reply:
I'll give it a try!

Reply:
So `${String()}` doesn't work (same with a few variations thereof which I tried), but I created an empty string in a variable and expanded the comment, see faf5a41.